neo4j-store/src/main/kotlin/org/ostelco/prime/storage/graph/Neo4jStore.kt

```diff
@@ -583,7 +583,13 @@ object Neo4jStoreSingleton : GraphStore {
                                                             iccId = simProfile.iccId,
                                                             status = simEntry.status,
                                                             eSimActivationCode = simEntry.eSimActivationCode,
-                                                            alias = simProfile.alias)
+                                                            alias = simProfile.alias,
+                                                            requestedOn = simProfile.requestedOn,
```

Comment:
Another thing is that this method is long, and is also highly indented. Both of these factors makes it less simple to read.  Using a builder pattern will make the method shorter, which is good, but also taking measures to reduce indentation levels would increase readability.